plugins/pa11y/test/pa11y.test.ts

```diff
@@ -21,12 +21,31 @@ jest.mock('@dotcom-tool-kit/logger')
 jest.mock('@dotcom-tool-kit/state')
 
 describe('pa11y', () => {
-  it("sets process.env.TEST_URL if readState('review') is truthy", async () => {
-    jest.mocked(state.readState).mockReturnValue({ appName } as state.ReviewState)
-
+  let MOCK_ENV: 'local' | 'ci' = 'local'
+  beforeEach(() => {
+    jest.mocked(state.readState).mockImplementation((stateType) => {
+      let returnState
+      if (stateType === 'local' && MOCK_ENV === 'local') {
+        returnState = { port: 5050 } as state.LocalState
+      }
+      if (stateType === 'review' && MOCK_ENV === 'ci') {
```

Comment:
**nitpick**: this could also use `else if` as they're mutually exclusive 😄

```suggestion
      } else if (stateType === 'review' && MOCK_ENV === 'ci') {
```